### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8545,10 +8545,15 @@ universtiy->university
 univesities->universities
 univesity->university
 unknonw->unknown
+unknonws->unknowns
 unknow->unknown
+unknowns->unknowns
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: This is identical

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: fixed. see 263a1be

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8552,10 +8552,15 @@ universtiy->university
...
-unknow->unknown
+unknonws->unknowns
+unknow->unknown, unknow,
+unknows->unknowns, unknows,
 unknwon->unknown
+unknwons->unknowns
 unkonwn->unknown
+unkonwns->unknowns
 unkown->unknown
+unkowns->unknown
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: this is likely also meant to be `unknowns`

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: indeed! apologies and thanks.

**Pull request merged**